src/components/RightSidebar.tsx

```diff
@@ -14,40 +14,45 @@ export type RightSidebarProps = {
   content: SidebarContentProps[];
 };
 
+/** Exporting this value allows layout components to coordinate responsive column sizing
+ * while avoiding layout shift when the sidebar is opened */
+export const RIGHT_SIDEBAR_MIN_WIDTH = "250px";
+
 export function RightSidebar({ content }: RightSidebarProps) {
   const [selectedIcon, setSelectedIcon] = useState<IconKey | undefined>(undefined);
   const tid = useTestIds({}, "rightSidebar");
-  const width = 380;
 
   return (
     <>
-      <div css={Css.df.jcfe.relative.pr3.$}>
```

Comment:
The design inspiration for this sidebar component come from here: https://dribbble.com/shots/25855464-Financial-Dashboard-Design
This change brings it closer to that intent where the icons are initially vertically positioned when the sidebar is closed, then animate back in horizontally at the top when the sidebar opens